inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -356,7 +356,7 @@ def execute_from_manifest(self,
                     self._update_logger.status = OTA_PENDING
                     self._update_logger.error = ""
                 self._update_logger.save_log()
-                if self.sota_mode == 'download-only' or self._device_reboot in ["No", "N", "n", "no", "NO"]:  # pragma: no cover
+                if self.sota_mode == 'download-only' or not self._is_reboot_device():
```

Comment:
The `execute_from_manifest` method should use the `_is_reboot_device` method to determine if a reboot is necessary, instead of directly checking the `self._reboot_device` variable. [important]

Reply:
That method isn't using that variable.